kolibri/plugins/management/assets/src/device_management/state/mutations/managePermissionsMutations.js

```diff
@@ -0,0 +1,7 @@
+export function SET_PERMISSIONS_PAGE_STATE(state, pageState) {
+  state.pageState = pageState;
+}
+
+export function SET_USER_PERMISSIONS_PAGE_STATE(state, pageState) {
+  state.pageState = pageState;
+}
```

Comment:
`SET_PAGE_STATE`, `SET_CONTENT_PAGE_STATE`, `SET_PERMISSIONS_PAGE_STATE`, and `SET_USER_PERMISSIONS_PAGE_STATE` are all identical. Curious about the motivation for defining them independently?

Reply:
These mutations used to have guards to make sure that `state.pageName` was correct before committing the mutation, but I removed them to use ConditionalPromise and place the guards at the `show*page` actions.

The value in keeping it this way is that it reads better inside of vue devtools.